src/qtism/data/storage/xml/Utils.php

```diff
@@ -284,7 +284,7 @@ public static function getDOMElementAttributeAs(DOMElement $element, string $att
 
         switch ($datatype) {
             case 'string':
-                return $attr;
+                return htmlspecialchars_decode($attr);
```

Comment:
Since this code will decode all the attributes now, I am curious if there is any case where we, on purpose, would like to keep the attribute value encoded.

Maybe would be nice to check with @julien-sebire too.

Reply:
Fixed by the new approach where we are not double encoding the attributes as it is implicitly performed by PHP through DOMElement.